Build error, maybe involving `Rule` and `ast::Rule` conflict?

Hi all. Getting an error on a fresh build of egglog on GitHub Actions runners. Weirdly, cannot replicate on my own machine even after clean. Using stable Rust on my machine.

```
Compiling egglog v0.1.0 (https://github.com/egraphs-good/egglog?rev=41562802315eccc9136c9d2ae6967d2df4071420#41562802)
error[E0560]: struct `Rule` has no field named `head`
   --> /root/.cargo/git/checkouts/egglog-0aabc6dbd51d8ac4/4156280/src/terms.rs:391:13
    |
391 |             head: self.instrument_actions(&rule.head),
    |             ^^^^ `Rule` does not have this field
    |
    = note: available fields are: `query`, `program`, `matches`, `times_banned`, `banned_until`, `todo_timestamp`

error[E0560]: struct `Rule` has no field named `body`
   --> /root/.cargo/git/checkouts/egglog-0aabc6dbd51d8ac4/4156280/src/terms.rs:392:13
    |
392 |             body: self.instrument_facts(&rule.body),
    |             ^^^^ `Rule` does not have this field
    |
    = note: available fields are: `query`, `program`, `matches`, `times_banned`, `banned_until`, `todo_timestamp`

error[E0308]: mismatched types
    --> /root/.cargo/git/checkouts/egglog-0aabc6dbd51d8ac4/4156280/src/terms.rs:397:13
     |
397  |             rule,
     |             ^^^^ expected `ast::Rule`, found `Rule`
     |
     = note: `Rule` and `ast::Rule` have similar names, but are actually distinct types
note: `Rule` is defined in module `crate` of the current crate
    --> /root/.cargo/git/checkouts/egglog-0aabc6dbd51d8ac4/4156280/src/lib.rs:423:1
     |
423  | struct Rule {
     | ^^^^^^^^^^^
note: `ast::Rule` is defined in module `crate::ast` of the current crate
    --> /root/.cargo/git/checkouts/egglog-0aabc6dbd51d8ac4/4156280/src/ast/mod.rs:1244:1
     |
1[244](https://github.com/uwsampl/lakeroad/actions/runs/7073200166/job/19252861215#step:4:245) | pub struct Rule {
     | ^^^^^^^^^^^^^^^
```

--- a/src/terms.rs
+++ b/src/terms.rs
@@ -1,4 +1,12 @@
-use crate::*;
+use crate::{
+    ast::{
+        desugar::Desugar, Action, Command, CommandId, Expr, Fact, NCommand, NormAction,
+        NormCommand, NormExpr, NormFact, NormFunctionDecl, NormRule, NormSchedule, Rule, RunConfig,
+        Schedule,
+    },
+    util::ListDisplay,
+    ArcSort, EGraph, Error, Symbol, TypeInfo,
+};
 
 /// The state of the term encoding is simply a current context
 /// so that it can look up type information.

